dashboard-api/docker-compose-test.yaml

```diff
@@ -0,0 +1,41 @@
+services:
```

Comment:
Ideally, we will specify the file version of docker-compose as the first line of this file, for example `version: '3'`, for more info see https://docs.docker.com/compose/compose-file/compose-file-v3/